packages/functional-tests/tests/postVerify/forcePasswordChange.spec.ts

```diff
@@ -2,42 +2,41 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { test, expect } from '../../lib/fixtures/standard';
-let email;
-const password = 'password';
-const newPassword = 'new_password';
+import {
+  test,
+  expect,
+  password,
+  newPassword,
+} from '../../lib/fixtures/standard';
+
 let emailUserCreds;
 
 test.describe('severity-2 #smoke', () => {
   test.describe('post verify - force password change', () => {
-    test.beforeEach(async ({ target, pages: { login } }) => {
+    test.use({
+      emailOptions: [
+        {
+          prefix: 'forcepwdchange{id}',
+          password: 'passwordzxcv',
+          newPassword: 'new_password',
```

Comment:
We can reuse the `import`ed values from L8-9 above:
```suggestion
          password,
          newPassword,
```

Reply:
incorporated. Thanks!